cmd/maya-apiserver/cstor-operator/spc/handler.go

```diff
@@ -23,7 +23,10 @@ import (
 
 	"github.com/golang/glog"
 	apis "github.com/openebs/maya/pkg/apis/openebs.io/v1alpha1"
+	apiutil "github.com/openebs/maya/pkg/apiutil"
+	apiutils "github.com/openebs/maya/pkg/apiutil"
```

Comment:
nit : `apituils` or `apiutil` either one ?

Reply:
Thanks I will remove it